spec/models/form/sales/questions/mortgage_amount_spec.rb

```diff
@@ -46,4 +46,20 @@
   it "has correct min" do
     expect(question.min).to be(1)
   end
+
+  context "when the mortgage is not used" do
+    let(:log) { create(:sales_log, :completed, mortgageused: 2, deposit: nil) }
```

Comment:
```suggestion
    let(:log) { build(:sales_log, :completed, mortgageused: 2, deposit: nil) }
```
It is my understanding that db actions are often the most expensive actions by time in rails. I believe we should avoid committing things to db in tests where it isn't necessary.